scss/atoms/input_styles/_input_styles.scss

```diff
@@ -0,0 +1,121 @@
+/*
+* ==========================================================================
+* Input field styles
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
```

Comment:
empty file, remove this and the file for now, add later when there is something to do in functions file